.automation/init-db.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+# Execute SQL commands
+psql -v ON_ERROR_STOP=1 --username "postgres" <<-EOSQL
+    CREATE USER ovirt PASSWORD 'ovirt';
```

Comment:
I guess this will trigger security alerts.
Can you also add a `.gitleaks.toml` file as in https://github.com/oVirt/ovirt-site/blob/main/.gitleaks.toml ?